src/main/java/frc/robot/Robot.java

```diff
@@ -63,11 +66,17 @@ public void teleopInit() {
 
   @Override
   public void robotPeriodic() {
-    yaw = yawProvider.getYaw();
-    CommandScheduler.getInstance().run();
+    if (robotPlatform.isPresent()) {
+      robotPlatform.get().robotPeriodic();
+    } else {
+      CommandScheduler.getInstance().run();
```

Comment:
Command scheduler should be run unconditionally

Reply:
I don't know what to do with that suggestion. Can you be specific with what you would change?